drivers/timer/Kconfig

```diff
@@ -169,6 +169,13 @@ config RV32M1_LPTMR_TIMER
 	  peripheral as the system clock. It provides the standard "system clock
 	  driver" interfaces.
 
+config ITE_IT8XXX2_TIMER
+	bool "ITE it8xxx2 timer driver"
+	depends on SOC_IT8XXX2
+	help
+	  This module implements a kernel device driver for the ITE it8xxx2
+	  HW timer model
```

Comment:
Can you add a few more details? Does it have a configurable frequency? Is it a 16/32-bit timer?...